revolutionpi/revolution_pi.go

```diff
@@ -154,5 +154,31 @@ func (b *revolutionPiBoard) DoCommand(ctx context.Context,
 		resp["getStatus"] = "status ok"
 	}
 
+	pinName, okAddr := req["readAddress"].(string)
```

Comment:
@thegreatco should I add a doCommand for writes too? not sure if I want to since letting users write to memory addresses might let them think they can make configuration changes that way